data_steward/utils/sandbox.py

```diff
@@ -63,3 +89,76 @@ def check_and_create_sandbox_dataset(project_id, dataset_id):
     if sandbox_dataset not in datasets:
         create_sandbox_dataset(project_id, dataset_id)
     return sandbox_dataset
+
+
+def get_sandbox_labels_string(src_dataset_name,
+                              class_name,
+                              table_tag,
+                              shared_lookup=False):
+    """A helper function that formats a set of labels for BigQuery
+
+    :param str src_dataset_name: A dataset name
+    :param str class_name: A class name
+    :param str table_tag: A table tag
+    :param bool shared_lookup: A boolean that indicates if a table is a shared lookup, defaults to False
+    :raises ValueError: Raises if one of first three arguments is whitespace or None
+    :raises ValueError: Raises if the `shared_lookup` is not a boolean
+    :return: A formatted string of the arguments concatenated as labels
+    :rtype: str
+    """
+    STRING_LABELS = ['src_dataset', 'class_name', 'table_tag']
+    BOOL_LABELS = ['shared_lookup']
```

Comment:
nit: maybe make this lowercase since they are instance variables

Reply:
Sounds good